src/warnet/control.py

```diff
@@ -157,17 +157,21 @@ def get_active_network(namespace):
 
 
 @click.command(context_settings={"ignore_unknown_options": True})
-@click.argument("scenario_file", type=click.Path(exists=True, file_okay=True, dir_okay=False))
+@click.argument("file", type=click.Path(exists=True, file_okay=True, dir_okay=False))
 @click.argument("additional_args", nargs=-1, type=click.UNPROCESSED)
-def run(scenario_file: str, additional_args: tuple[str]):
-    """Run a scenario from a file"""
-    scenario_path = os.path.abspath(scenario_file)
-    scenario_name = os.path.splitext(os.path.basename(scenario_path))[0]
+def run(file: str, additional_args: tuple[str]):
+    """Run a scenario or other executable"""
+    file_path = os.path.abspath(file)
+    file_name = os.path.splitext(os.path.basename(file_path))[0]
 
-    with open(scenario_path) as file:
-        scenario_text = file.read()
+    import base64
 
-    name = f"commander-{scenario_name.replace('_', '')}-{int(time.time())}"
+    with open(file_path, "rb") as f:
+        b_64_file = base64.b64encode(f.read())
+
+    pull_policy = "Never" if os.getenv("GITHUB_ACTIONS") else "IfNotPresent"
```

Comment:
why can't you set `pullPolicy: Never` on the node-defaults.yaml for the test?

Reply:
ah, I see, its because the commander is not deployed via helm 😢 (and node-defaults refers to just tanks, anyways).

I think we need to fix the commander first and have it be deployed via a helm chart, or at least move the k8s yaml out of the python code and decided how we want to configure it. Then, we can rebase this on that PR? 

I don't think having our python code be aware it's running in CI and then changing behaviour based on that is a good idea. Let me know if you have any interest in moving the commander to a helm chart, or I can take a crack at it once I've finished the signet stuff.

Reply:
Attempted in #550 👍🏼

Reply:
Now that 550 is in can this be revisited?